docs/UPGRADE-3.7.0.md

```
@@ -0,0 +1,53 @@
+```hcl
+# Multiple instance types for main node groups
+lin_instance_type = "m5.large"
+win_instance_type = "t3.xlarge"
+
+# If you set both variables, *_instance_type_list takes precedence over *_instance_type
+lin_instance_type_list = ["m5.large", "m5.xlarge", "m6i.large"]
+win_instance_type_list = ["t3.xlarge", "t3.2xlarge"]
+
+```
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)